src/utils/DateUtils.ts

```diff
@@ -11,6 +11,15 @@ class DateUtils {
 		if (days === 1) return "yesterday";
 		return `${days} days ago`;
 	}
+
+	static format(date: Date) {
```

Comment:
I feel like ”format” is quite a generic name, maybe something like formatAsText?